FrontEnd/src/index.scss

```diff
@@ -1,23 +1,21 @@
-@import './colors.scss';
+@import "./colors.scss";
```

Comment:
I've added a sass formatter (sibiraj-s.vscode-scss-formatter) extention to VSCode, and that thinks these should be double =$

Reply:
fair enough haha -- i wasn't too sure when i did it either!!